plugin-server/src/worker/ingestion/team-manager.ts

```diff
@@ -39,7 +48,45 @@ export class TeamManager {
         }
     }
 
-    public async updateEventNamesAndProperties(teamId: number, event: string, properties: Properties): Promise<void> {
+    async flushLastSeenAtCache(): Promise<void> {
```

Comment:
Can we log this to the console, together with e.g. nr of events, total elapsed time for the flush, etc.

Reply:
Yessir